lib/passwordless/url_safe_base_64_generator.rb

```diff
@@ -1,7 +1,11 @@
 # frozen_string_literal: true
 
 module Passwordless
+  # Generates secure random numbers for sessions/cookies, etc.
   class UrlSafeBase64Generator
+    # Generates a url safe base64 secure random number. :-)
+    # @param _session [Object] some Object that contains this class.
```

Comment:
@mikker  I wasn't sure if I fully understood how to use this param. Maybe there is a better way to describe it?

Reply:
We pass the session to the generator so that if you wanted to use it in any way in your custom token generator you can. Say your user has a name you could make a custom generator that uses that for the tokens:

```ruby
class NameTokenGenerator
  def self.call(session)
    "#{session.authenticatable.name}-gets-a-token"
  end
end
```

Happens here:
https://github.com/mikker/passwordless/blob/master/app/models/passwordless/session.rb#L25